### DISCUSSION TITLE
ffmpeg doesn't work in Github codespace

### DISCUSSION THREAD

**Discussion by dauitsuragan002** [community/community#131449] [Codespaces]
### Select Topic Area

Question

### Body

![image](https://github.com/community/community/assets/78667630/045db8e8-1145-484c-ba6c-c9377a2b11e7)

Мен шешімін таптым рақмет шәке ChatGPT:

sudo apt update
sudo apt install ffmpeg

**Comment by Akash1134** [comment]
Closing as resolved
